src/pages/Contributors/AffiliatedOrganizations.js

```diff
@@ -66,65 +66,84 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-export const AffiliatedOrganizations = ({ organizations , inputValue, data, checkboxValue }) => {
-
+export const AffiliatedOrganizations = ({
+  organizations,
+  inputValue,
+  organizationData,
+  checkboxValue,
+}) => {
   const classes = useStyles();
-  const parentfilterData = data;
-  const [isChildThumbnail] = useState(true);
+  const parentfilterData = organizationData;
+  const isChildThumbnail = true;
+  let organizationArray;
+  let parentdata;
+  let parentChildobj;
+  let mapsearchedChildParent;
+
+  const sortOn = (sortingObject) => {
+    return function (a, b) {
+      return a[sortingObject]
+        .toLowerCase()
+        .localeCompare(b[sortingObject].toLowerCase());
+    };
+  };
 
   const getParentData = () => {
-    const parentOrg =organizations['Code for All'].filter(item => item);
-    const parentdata = [];
+    organizationArray = organizations['Code for All'].filter((item) => item);
+
+    parentdata = [];
 
-    parentOrg.forEach((data)=>{
-      if (data.depth === 3){
-        data['childNodes'] = [];
-        data['isOpen'] = false;
-        parentdata.push(data);
+    organizationArray.forEach((organizationData) => {
```

Comment:
You have used the variable name 'organizationData' inside the loop but it is also the name of a prop. Do not have two things with the same name. 

Better to use a short name here, like 'org.'

```
organizationArray.forEach((org) => {
etc
```